example/spec/parallel/discoverGranules/DiscoverGranulesWithExecutionNamePrefixSpec.js

```diff
@@ -97,22 +107,32 @@ describe('The DiscoverGranules workflow', () => {
   });
 
   afterAll(async () => {
+    await Promise.all(
+      queueGranulesOutput.payload.running
```

Comment:
I noted while re-reviewing -- If the purpose of waiting for completed execution is simply to delete the final granule write, we should probably be using the executions API to wait for completion instead of the step functions API (which this helper ultimately uses), *then* check for granule status, *then* delete, if we want to be sure we're using the database state to determine status.